scripts/pocket-change.js

```diff
@@ -39,17 +40,21 @@ export default class PocketChange {
    *
    * @param {Actor5e} actor - The actor to base the coin generation off of
    */
-  generateCurrency(actor) {
+  generateCurrency(actor, ignoreRating) {
     let currency;
 
-    if (this._isWithinChallengeRating(actor, 0, 4)) {
-      currency = this._treasureForChallengeRating0to4(actor);
-    } else if (this._isWithinChallengeRating(actor, 5, 10)) {
-      currency = this._treasureForChallengeRating5to10(actor);
-    } else if (this._isWithinChallengeRating(actor, 11, 16)) {
-      currency = this._treasureForChallengeRating11to16(actor);
-    } else if (this._isWithinChallengeRating(actor, 17, 100)) {
-      currency = this._treasureForChallengeRating17andUp(actor);
+    if(String(ignoreRating) === 'true') {
```

Comment:
What's the point of stringifying it first? Does this protect against something?

Reply:
Yes against  my stupidity, but in this case i just set the boolean check